### PR TITLE
Initial UI Commit

### PR DESCRIPTION
**Pull request by ghost**
Initial Commit Work

### CODE REVIEW

**Comment by michaeljones** [comment]
Do you have a preference for `yarn` over `npm`? I do personally but I didn't know what we might pick.

**Comment by michaeljones** [comment]
I would have a preference for typescript if you're going to put the work into PropTypes. I don't know how the group feels about it though?

**Comment by michaeljones** [comment]
Looks great generally. Love the screenshot in the chat thread.

File: frontEnd/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import EmojiButton from "./EmojiButton/EmojiButton";
+import styles from "./App.module.css";
+
+function App() {
+  const handleButtonPress = label => {
+    updateSelectedMood(label);
+  };
+
+  const emojiButtonArray = [
+    { label: "🤠" },
```
Comment (michaeljones) [inline frontEnd/src/components/App.jsx:12]: I realise that unicode characters are legitimate strings but I'd had a preference for storing "happy" or "sad" or "anxious" or something that is a bit more readable for everyone.

File: frontEnd/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    { label: "🤑" },
+    { label: "🤧" },
+    { label: "🥳" },
+    { label: "😭" }
+  ];
+
+  const [selectedMood, updateSelectedMood] = useState("");
```
Comment (michaeljones) [inline frontEnd/src/components/App.jsx]: Very minor but I'd prefer "null" over an empty string as the empty value but it is pretty subjective stuff.

File: frontEnd/src/components/EmojiButton/EmojiButton.module.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.EmojiButton {
+  height: 100px;
+  width: 100px;
+  margin: 20px;
+  background-color: lightgray;
```
Comment (michaeljones) [inline frontEnd/src/components/EmojiButton/EmojiButton.module.css:5]: Could optionally collect the colours in css variables in a `:root` block somewhere for easy reuse. Again, very minor.

**Review by michaeljones — APPROVED** [review]
Great stuff. Never seen the web vitals before. Looking forward to learning more about that.

**Review by tinwilco — APPROVED** [review]

**Pull request merged**